A6-client-retries.md

```diff
@@ -106,9 +106,9 @@ gRPC's call deadline applies across all attempts for a given RPC. For example, i
 
 ##### Exponential Backoff
 
-The `initialBackoff`, `maxBackoff`, and `backoffMultiplier` parameters determine the randomized delay before retry attempts.
+The `initialBackoff`, `maxBackoff`, and `backoffMultiplier` parameters tune the exponential delay before retry attempts.
 
-The initial retry attempt will occur at `random(0, initialBackoff)`. In general, the `n`-th attempt will occur at `random(0, min(initialBackoff*backoffMultiplier**(n-1), maxBackoff))`.
+The initial retry attempt will occur after `initialBackoff`.  After that, the `n`-th attempt will occur after `min(initialBackoff*backoffMultiplier**(n-1), maxBackoff) * random(0.8, 1.2))`.
```

Comment:
Subtle: the jitter is applied to the maxBackoff. I expect you thought about that when writing it. I think the idea is "Yes, the backoff can exceed maxBackoff, but that confusing part is better than not having the jitter." I'm willing to agree, but wanted to call out the detail. (And I don't think we need to discuss the option of doing 0.8-1x the maxBackoff.)

Reply:
That hadn't actually occurred to me, but I agree with you that it's the right behavior.  Even when we get to max backoff, we want the jitter, and I don't see any reason to cap it that strictly.

This also matches the [connection backoff spec](https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md), which keeps everything consistent.